src/test/java/uk/gov/hmcts/reform/sendletter/services/LetterServiceTest.java

```diff
@@ -64,7 +71,14 @@ public void should_return_message_id_when_single_letter_is_sent() throws Excepti
 
         verify(queueClientSupplier).get();
         verify(queueClient).sendAsync(any(Message.class));
-        verifyNoMoreInteractions(queueClientSupplier, queueClient);
+
+        voidCompletableFuture.thenRun(() -> {
+            verify(queueClient).closeAsync();
+            verify(insights).trackMessageAcknowledgement(messageId);
+            verify(insights).trackMessageReceived("service", letter.template, messageId);
+            verifyNoMoreInteractions(queueClientSupplier, queueClient);
```

Comment:
Do we need both of these ?

Reply:
if you check for interaction of mocks you have in one test case scope - best to include all of them. in fact, i'm missing `insights` there. probably after investigation as to why the tests did not work on travis